### PR TITLE
Add listener notification when not licensed

### PR DESCRIPTION
**Pull request by andrecardoso**
- Handle NOEXIST message to notify authorization status listeners and fix client info log

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(2) ChrOS: Chrome OS Player shows as unsupported player](https://trello.com/c/Ge1NElSX/4632-2-chros-chrome-os-player-shows-as-unsupported-player)

File: src/licensing/index.js [hunk 1]
```diff
@@ -59,14 +68,17 @@ function submitWatchForProductAuthChanges() {
...
+  if (status !== 'CURRENT' && status !== 'NOEXIST') {return}
   if (!aProductAuthFileWasChanged()) {return}
 
-  const productCode = Object.values(productCodes).find(prodCode=>{
+  const productCode = products().find(prodCode=>{
     return filePath.includes(prodCode);
   });
 
-  return fileSystem.readCachedFileAsObject(ospath.split("/").pop())
+  const objectPromise = status === 'CURRENT' ? fileSystem.readCachedFileAsObject(ospath.split("/").pop()) : Promise.resolve({productCode: false});
```
Comment (tejohnso) [inline src/licensing/index.js]: I think this should be `Promise.resolve({[productCode]: false})`

Comment (andrecardoso) [reply src/licensing/index.js]: Yes!

**Comment by andrecardoso** [comment]
@tejohnso could you please re-review? Thanks

**Pull request merged**